docs/projects/even-odd/_posts/2018-12-31-python.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Even-Odd in Python
+layout: default
+last-modified: 2018-12-31
+tags: ['python', 'even-odd', 'syntax']
+authors: "Chase Stevens"
```

Comment:
Hey @chase-stevens! Can you add yourself to the [authors.yml](https://github.com/TheRenegadeCoder/sample-programs/blob/master/docs/_data/authors.yml) file (it's alphabetical)? Whatever username you decide goes here.